modules/culling/src/lib/bounding-volumes/oriented-bounding-box.ts

```diff
@@ -98,6 +134,178 @@ export class OrientedBoundingBox implements BoundingVolume {
     return this;
   }
 
+  /**
+   * Computes an OrientedBoundingBox that bounds a region on the surface of the WGS84 ellipsoid.
+   * There are no guarantees about the orientation of the bounding box.
+   *
+   * @param {number[]} region The cartographic region ([west, south, east, north, minimum height, maximum height])
+   *                          on the surface of the ellipsoid.
+   * @returns {OrientedBoundingBox} The modified result parameter or a new OrientedBoundingBox instance if none was provided.
+   */
+  // eslint-disable-next-line max-statements
+  fromRegion(region: number[]): OrientedBoundingBox {
```

Comment:
Nit: From tree shaking (bundle size) perspective, it would be preferable to offer this as a global function rather than a method on the OBB class.  That way we could also avoid this file from growing too big.